content/en/platform/corda/1.3/cenm/auth-service.md

```diff
@@ -75,7 +75,7 @@ To do this:
 2. Generate a `jwt` signing key (RSA keypair) in a jks file with the following command line command:
 `keytool -genkeypair -alias mytest -keyalg RSA -keypass mypass -keystore mytest.jks -storepass mypass`.
 
-3. Ensure you have your A baseline `.jar` file that contains the set of permissions available in the deployment and optionally predefined roles. This is shipped as part of CENM via Artifcatory. For example, the file for CENM 1.3 is called `accounts-baseline-cenm-1.3.0.jar`. You can also copy this into a folder inside the working directory called `plugins` to avoid having to specify it in the config file.
+3. Ensure you have your A baseline JAR file that contains the set of permissions available in the deployment and optionally predefined roles. This is shipped as part of CENM via Artifcatory. For example, the file for CENM 1.3 is called `accounts-baseline-cenm-1.3.0.jar`. You can also copy this into a folder inside the working directory called `plugins` to avoid having to specify it in the config file.
```

Comment:
```suggestion
3. Ensure you have a baseline JAR file that contains the set of permissions available in the deployment and optionally predefined roles. This is shipped as part of CENM via Artifcatory. For example, the file for CENM 1.3 is called `accounts-baseline-cenm-1.3.0.jar`. You can also copy this into a folder inside the working directory called `plugins` to avoid having to specify it in the config file.
```

Reply:
I know not directly related to your updates but jumped out at me

Reply:
Fixed.